### DISCUSSION TITLE
Should we check in `go.work`?

### DISCUSSION THREAD

**Discussion by joschahenningsen** [TUM-Dev/gocast#804] [General]
We recently removed the go workspace file from our repo arguing that is is only intended for local specific use. 

This discussion mentions 
> We could recommend never checking them in, but I think that's a bad idea: in some particular scenarios it can make perfect sense to check them in, such as monorepos holding multiple modules.

I'd argue this applies to us because we combine multiple applications in our repo (TUM-Live, Worker, Edge,...)  that depend on each other. Especially updates to protobuf definitions have to be separated into two PRs with our current approach.

What do you think? @alexanderstephan @MatthiasReumann @mono424

**Comment by joschahenningsen** [comment]
In #582 we reintroduced the workspace to the repo, I suggest leaving it in if it doesn't cause trouble.

**Reply by alexanderstephan** [reply]
Sounds good! :+1: We still can keep an eye on the [issue](https://github.com/golang/go/issues/53502) whether there is an official recommendation, but for now it seems like the best thing to do.
